Integrations/Cloaken/Cloaken.py

```diff
@@ -70,3 +73,39 @@ def get_client():
     else:
         # server error or unavailable
         return_error('Error Cloaken Unshorten: ' + str(response.get('data', 'key missing')))
+
+if demisto.command() == 'cloaken-screenshot-url':
+    client = get_client()
+    url = demisto.args()["url"]
+    screenshot = utility.RasterizeAndRetrieveImage(client, url=url)
+    context={"Url":url, "Status":'failed'}
+    try:
+        result = screenshot.get_screenshot()
+        stored_img = fileResult(result.get("filename","screenshot.png"), result.get("data","NA"), "image")
```

Comment:
verify `data` exists in the result instead of using default of `NA`, as `NA` is not a valid png file content

Reply:
Fixed